requirements/dev.txt

```diff
@@ -5,41 +5,55 @@
 #    pip-compile --output-file requirements/dev.txt requirements/base.in requirements/dev.in requirements/quality.in
 #
 argparse==1.4.0           # via caniusepython3
-backports.functools-lru-cache==1.5  # via caniusepython3
-caniusepython3==6.0.0
+astroid==1.5.2            # via edx-lint, pylint, pylint-celery
+backports.functools-lru-cache==1.5  # via astroid, caniusepython3, pylint
+caniusepython3==7.0.0
 certifi==2018.4.16        # via requests
 chardet==3.0.4            # via requests
-click==6.7                # via pip-tools
-configparser==3.5.0       # via pydocstyle
-diff-cover==1.0.2
+click-log==0.1.8          # via edx-lint
+click==6.7                # via click-log, edx-lint, pip-tools
+configparser==3.5.0       # via pydocstyle, pylint
+diff-cover==1.0.4
 distlib==0.2.7            # via caniusepython3
-django==1.9.8
+django==1.11.15
```

Comment:
Why Django 1.11.15 and not Django 2 ?

Reply:
Good question - this particular requirements in `dev.py` is only for bringing up the `test` django package, and doesn't impact the support of the project. It's specified in [`base.in`](https://github.com/HBS-HBX/django-elastic-migrations/pull/4/files#diff-d9383e4fc78d9b72886e207884f8f14a), which, when `make update` is called should autoupdate to the latest version available. I'll need to research which requirement is holding this one back. Shouldn't particularly matter, but we don't yet have tests for Django 2 - that's TBD. I'll create an issue for it.

Reply:
created GH issue #6 for testing support for django 2